src/NuGet.Core/NuGet.Protocol/Plugins/PluginDiscoveryUtility.cs

```diff
@@ -0,0 +1,59 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using NuGet.Common;
```

Comment:
nit : sort usings

Reply:
they are sorted? 

Internal namespaces always come after the system namespaces.